### DISCUSSION TITLE
Unable to init the project with go mod tidy v0.16.0

### DISCUSSION THREAD

**Discussion by daviFrag** [99designs/gqlgen#1854] [Q&A]
### What happened?
While i was reinitializing the go.mod file in my project go outputs this error:
````
go: found github.com/99designs/gqlgen/example/federation/accounts/graph/model in github.com/99designs/gqlgen/example v0.0.0-20220125191136-c161ab382547
go: found github.com/99designs/gqlgen/example/federation/products/graph/model in github.com/99designs/gqlgen/example v0.0.0-20220125191136-c161ab382547
go: found github.com/99designs/gqlgen/example/federation/reviews/graph/model in github.com/99designs/gqlgen/example v0.0.0-20220125191136-c161ab382547
go: github.com/99designs/gqlgen/example@v0.0.0-20220125191136-c161ab382547 requires
        github.com/99designs/gqlgen@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
````

### What did you expect?
If i manually install the gqlgen@v0.14.0 it works perfectly

### versions
 - `gqlgen version`? v0.16.0
 - `go version`? 1.16
 - dep or go modules? go modules

**Comment by ipfans** [comment]
Can you attach your go.mod?

**Comment by daviFrag** [comment]
My go.mod file is empty, when i try to run the go mody tidy command it throws the error described. I also include the tools.go in my project as described in the doc

```
module github.com/username/name

go 1.16

```

**Comment by ipfans** [comment]
Oh, can you try these steps?

```
go get github.com/99designs/gqlgen@latest
go mod tidy
```

And can you show which gqlgen version of this project before you try to reinitialize it?

**Comment by daviFrag** [comment]
I tried this steps but i've got the same result. The previous version of the gqlgen was v0.14.0. I noticed that even with the v0.15.0 i have the same result.

The mod tidy works as intended if i run:
```
go get github.com/99designs/gqlgen@v0.14.0
go mod tidy
```

This is the correct go.mod generated:
```
module github.com/username/name

go 1.16

require (
	github.com/99designs/gqlgen v0.14.0
	github.com/go-redis/redis/v8 v8.11.4
	github.com/gobwas/ws v1.1.0
	github.com/golang-jwt/jwt v3.2.2+incompatible
	github.com/google/uuid v1.3.0
	github.com/jensneuse/abstractlogger v0.0.4
	github.com/jensneuse/graphql-go-tools v1.43.2
	github.com/jensneuse/graphql-go-tools/examples/federation v0.0.0-20220124090656-47621af0e283 // indirect
	github.com/vektah/gqlparser/v2 v2.2.0
	go.uber.org/zap v1.20.0
	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
	gorm.io/driver/postgres v1.2.3
	gorm.io/gorm v1.22.5
)
```

**Comment by daviFrag** [comment]
I found that when i try to use go mod tidy with the last version, in my go directory the importer creates two different folders:
gqlgen->example@v0.0.0-20220125191136-c161ab382547
gqlgen@v0.16.0->all the folders of the project

edit: I'm on a windows machine

**Comment by pradeepkusingh** [comment]
Yes, same problem I am facing..

**Comment by softshipper** [comment]
Hi all,
I have the same problem and I could not figure out why it happens. When I run `go run github.com/99designs/gqlgen generate` it shows the following error message:

```
go run github.com/99designs/gqlgen generate
reloading module info
generating core failed: unable to load calshare.io/service/graph/model - make sure you're using an import path to a package that exists
```
and running `go mod tidy` it shows:
```
go: finding module for package calshare.io/service/graph/generated
go: finding module for package calshare.io/service/graph/model
calshare.io/service imports
        calshare.io/service/graph/generated: cannot find module providing package calshare.io/service/graph/generated: unrecognized import path "calshare.io/service/graph/generated": https fetch: Get "https://calshare.io/service/graph/generated?go-get=1": dial tcp: lookup calshare.io: no such host
calshare.io/service/graph imports
        calshare.io/service/graph/model: cannot find module providing package calshare.io/service/graph/model: unrecognized import path "calshare.io/service/graph/model": https fetch: Get "https://calshare.io/service/graph/model?go-get=1": dial tcp: lookup calshare.io: no such host
```

Would be great if someone can help.
[service.zip](https://github.com/99designs/gqlgen/files/7966753/service.zip)

**Reply by pradeepkusingh** [reply]
This seems different issue from the one reported initially. Seems coding and import issue.

**Reply by softshipper** [reply]
Could you please help. What am I doing wrong?

**Reply by frederikhors** [reply]
I answered you here https://github.com/99designs/gqlgen/issues/1860#issuecomment-1025221671.

**Comment by frederikhors** [comment]
@daviFrag https://github.com/99designs/gqlgen/pull/1872 fixes this.

Basically you can comment out `autobind` in `config.yml`.

**Reply by daviFrag** [reply]
The problem was issued is not about generation. I'm currently using gqlgen in a production backend setup as a federation (the autobind config is already commented out). The only way i can fix this importation issue is by creating a new go.mod file, then set manually the version v0.14.0 of gqlgen, run go mod tidy, and finally upgrade the version of gqlgen to v0.16.0.
if i delete all the dirs in the go package 99design folder, run go mod init  in the project and then go mod tidy, i face the problem again.

The real issue is generated when i try to use go mod tidy, in my go directory the importer creates two folders and that brakes the process:
gqlgen->example@v0.0.0-20220125191136-c161ab382547
gqlgen@v0.16.0->all the folders of the project

`go: github.com/99designs/gqlgen/example@v0.0.0-20220130014535-a30b68de58cb requires
        github.com/99designs/gqlgen@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000`

**Reply by frederikhors** [reply]
I'm sorry. I opened https://github.com/99designs/gqlgen/issues/1873.
